pkg/services/alerting/eval_handler_test.go

```diff
@@ -153,7 +153,8 @@ func TestAlertingEvaluationHandler(t *testing.T) {
 			So(context.ConditionEvals, ShouldEqual, "[[false OR false] OR false] = false")
 		})
 
-		Convey("Should retuasdfrn no data if one condition has nodata", func() {
+		// FIXME: What should the actual test case name be here?
+		Convey("Should retuasdfrn no data if one condition has no data", func() {
```

Comment:
```suggestion
		Convey("Should not return no data if no condition has no data and using OR", func() {
```

Reply:
@papagian Could we rewrite this more naturally as "Should return data if all conditions have data and using OR"? Good job on figuring out what this test does BTW!?

Reply:
yes, that's better

Reply:
@papagian Please just update the suggestion and I'll commit it :+1:

Reply:
on second thought though `context.NoDataFound` is boolean so it does not return data anyway

Reply:
@papagian In that case, maybe we should rephrase it as "Should not return NoData if all conditions have data and using OR"?